neuro_san/internals/utils/file_of_class.py

```diff
@@ -1,105 +0,0 @@
-
```

Comment:
This file is no longer needed.
We are now using the leaf-common version everywhere.